src/app/dao/tests/JoinerDAOMock.spec.ts

```diff
@@ -58,13 +66,16 @@ describe('JoinerDAOMock', () => {
         expect(callCount).toEqual(1);
         expect(lastJoiner.get()).toEqual(JoinerMocks.INITIAL);
 
-        await joinerDAOMock.update('joinerId', JoinerMocks.WITH_FIRST_CANDIDATE);
+        // When it is updated
+        await joinerDAOMock.update('joinerId', JoinerMocks.WITH_CHOSEN_OPPONENT);
 
+        // Then we should have seen the update
         expect(callCount).toEqual(2);
-        expect(lastJoiner.get()).toEqual(JoinerMocks.WITH_FIRST_CANDIDATE);
+        expect(lastJoiner.get()).toEqual(JoinerMocks.WITH_CHOSEN_OPPONENT);
         unsubscribe();
     }));
     it('Partial update should update', fakeAsync(async() => {
+        // Given an initial joiner to which we subscribed to the changes
```

Comment:
to which changes we subscribed to ? :D

Reply:
errr moins correct non ? auquel changements on a souscrit ?

Reply:
aux changements duquel on s'est abonné!
Là je lis "étant donné un document initial auquel on s'est abonné à ses changements. Après on peut se dire que "to the changes" est redondant, tu t'abonnes pas à la création du doc mais bien à ses changements ?

Reply:
vrai, je supprime "to the changes"